resources/views/components/highlight.blade.php

```diff
@@ -0,0 +1,12 @@
+@props(['attribute', 'item' => 'item'])
+
+@if (in_array($attribute, config('rapidez.searchkit.highlight_attributes')))
+    <ais-highlight
+        {{ $attributes }}
+        attribute="{{ $attribute }}"
+        :hit="{{ $item }}"
+        highlighted-tag-name="mark"
+    />
+@else
+    <span {{ $attributes }} v-text="{{ $item }}.{{ $attribute }}"></span>
+@endif
```

Comment:
- Currently it's not possible to change the tag
- The `highlighted-tag-name` default is `mark`, no need to specify it
- The `attribute` is an actual attribute, maybe a bit weird to have it as prop? 
- `item` does have a default, so I would go for everything (+ `attribute` with `name`) or nothing

Maybe something like this is better:

```suggestion
@props(['item' => 'item'])

@if (in_array($attribute, config('rapidez.searchkit.highlight_attributes')))
    <ais-highlight {{ $attributes->merge([
        'attribute' => 'name',
        ':hit' => $item,
    ]) }}/>
@else
    <span {{ $attributes }} v-text="{{ $item }}.{{ $attributes->get('attribute', 'name') }}"></span>
@endif
```

Reply:
What's the choice here between using `$attributes->get('attribute', 'name')` instead of just setting it as a prop?

We can do that option a bit cleaner with something like this (we can do without that bit of php at the start, but then you're using `$attributes->get('attribute', 'name')` 3 times if you want it to actually work):

```blade
@props(['item' => 'item'])

@php($attributes['attribute'] ??= 'name')
@if (in_array($attributes['attribute'], config('rapidez.searchkit.highlight_attributes')))
    <ais-highlight {{ $attributes->merge(['v-bind:hit' => $item]) }}/>
@else
    <span {{ $attributes }} v-text="{{ $item }}.{{ $attributes['attribute'] }}"></span>
@endif
```

But at that point I don't see why we shouldn't just keep it as a prop:

```blade
@props(['attribute' => 'name', 'item' => 'item'])

@if (in_array($attribute, config('rapidez.searchkit.highlight_attributes')))
    <ais-highlight {{ $attributes->merge([
        'v-bind:hit' => $item,
        'attribute' => $attribute,
    ]) }}/>
@else
    <span {{ $attributes }} v-text="{{ $item }}.{{ $attribute }}"></span>
@endif
```

This is exactly what Laravel props are for, allowing you to easily set default values and use them more liberally around the component. Especially because we don't need `attribute` directly set on the span anyway, I don't see why we should do it any other way.

Reply:
I also do find it a bit weird to have a default attribute here. Some resources may have a `title` instead of a name. It doesn't seem logical to have a default for that when it could be anything.

Reply:
Thanks for checking! 🚀